chain-impl-mockchain/src/block.rs

```diff
@@ -13,7 +14,7 @@ pub use crate::date::{BlockDate, BlockDateParseError};
 /// with the position of that block in the chain.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Block {
-    pub slot_id: BlockDate,
+    pub slot_id: BlockDate, // FIXME: rename to 'date'
```

Comment:
I'd rather comment "FIXME: hide all these public members and add accessors for them". But then it's mockchain at least for the time being.